### DISCUSSION TITLE
Welcome to AsyncFSWebServer_Teensy41 Discussions!

### DISCUSSION THREAD

**Discussion by khoih-prog** [khoih-prog/AsyncFSWebServer_Teensy41#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jakespeed1311** [comment]
HI,
Sorry if this is not the right place, but i want that you know that i have started. I am not familiar with git operations, at this moment.
First i opend an issui.. hmm

i have some troube to bring files t the Teensy , now i have add the Teensy MTP to the Sourcecode. To be able to copy teh files to to LittleFS.
From the PC and form Teensy Programm i can see that now. Hurray.

Next step is to check out why on my tests the website was not found from Async SerfeStatic on the FS.

greedings
Jake
